.gitignore

```diff
@@ -28,3 +28,6 @@ vendor/ruby
 
 # Generated js bundles
 /app/assets/javascripts/generated/*
+
+# Rubymine/IntelliJ
+.idea
```

Comment:
I'm happy to revert this - but can you explain the logic behind this one?  Moving things like this to a personal .gitignore file essentially decentralizes the logic of what to ignore and puts it on to each individual developer.  2 examples:

1 -  If someone comes along to contribute and they don't have a personal gitignore, they might accidentally commit files in .idea to this project.
2 - I notice you have .ruby-version in your global gitignore above. What if you're working a project that wants that committed? 

I'm not arguing the point here - I'm just curious as to the reasoning.